package.json

```diff
@@ -54,7 +54,7 @@
   "scripts": {
     "vscode:prepublish": "npm run package",
     "compile": "node scripts/build",
-    "watch": "npm run compile -- --watch",
```

Comment:
I think it's an intended change. Isn't it work for you? (because `npm run compile --watch` is not working for me)